dotnet-desktop-guide/net/winforms/printing/overview.md

```diff
@@ -0,0 +1,154 @@
+dev_langs:
+  - "csharp"
+  - "vb"
+ms.custom: devdivchpfy22
+---
+
+# PrintDialog Component Overview (Windows Forms .NET)
+
+Printing in Windows Forms consists primarily of using the [PrintDocument](../../../framework/winforms/controls/printdocument-component-windows-forms.md) component to enable the user to print. The [PrintPreviewDialog](../../../framework/winforms/controls/printpreviewdialog-control-windows-forms.md) control, [PrintDialog](../../../framework/winforms/controls/printdialog-component-windows-forms.md) and [PageSetupDialog](../../../framework/winforms/controls/pagesetupdialog-component-windows-forms.md) components provide a familiar graphical interface to Windows operating system users.
+  
+Typically, you create a new instance of the <xref:System.Drawing.Printing.PrintDocument> component and set the properties that describe what to print using the <xref:System.Drawing.Printing.PrinterSettings> and <xref:System.Drawing.Printing.PageSettings> classes. Call to the <xref:System.Drawing.Printing.PrintDocument.Print%2A> method actually prints the document.
+
+The Windows Forms [PrintDialog](../../../framework/winforms/controls/printdialog-component-windows-forms.md) component is a pre-configured dialog box used to select a printer, choose the pages to print, and determine other print-related settings in Windows-based applications. It's a simple solution for printer and print-related settings instead of configuring your own dialog box. You can enable users to print many parts of their documents: print all, print a selected page range, or print a selection. By relying on standard Windows dialog boxes, you create applications whose basic functionality is immediately familiar to users. The <xref:System.Windows.Forms.PrintDialog> component inherits from the <xref:System.Windows.Forms.CommonDialog> class.
+
+## Working with the Component
+
+Use the <xref:System.Windows.Forms.CommonDialog.ShowDialog%2A> method to display the dialog at run time. This component has properties that relate to either a single print job (<xref:System.Drawing.Printing.PrintDocument> class) or the settings of an individual printer (<xref:System.Drawing.Printing.PrinterSettings> class). One of the two, in turn, may be shared by multiple printers.
+
+ The show dialog box method helps you to add print dialog box to the form. The <xref:System.Windows.Forms.PrintDialog> component appears in the tray at the bottom of the Windows Forms Designer in Visual Studio.
+
+## How to: Capture User Input from a PrintDialog at Run Time
+
+You can set options related to printing at design time. Sometimes you may want to change these options at run time, most likely because of choices made by the user. You can capture user input for printing a document using the <xref:System.Windows.Forms.PrintDialog> and the <xref:System.Drawing.Printing.PrintDocument> components.  
+  
+### To change print options programmatically  
```

Comment:
Considering there isn't any other ### heading in this section, this is useless. Convert it into the intro to the procedure steps. Also, unless it's also going to demonstrate doing something non-programmatically, we can assume the reader is always doing something programmatically, so we can remove that word. I've actually just rewritten the heading as the intro because the wording of the heading doesn't even match the intent of the code.

```suggestion
You can set options related to printing at design time. Sometimes you may want to change these options at run time, most likely because of choices made by the user. You can capture user input for printing a document using the <xref:System.Windows.Forms.PrintDialog> and the <xref:System.Drawing.Printing.PrintDocument> components. The following steps demonstrate displaying the print dialog for a document:
```